e2e/test/scenarios/question-reproductions/reproductions-3.cy.spec.js

```diff
@@ -1584,47 +1585,40 @@ describe("issue 44668", () => {
 });
 
 // TODO: unskip when metabase#44974 is fixed
-describe.skip(
-  "issue 44974",
-  {
-    tags: ["@external"],
-  },
-  () => {
-    beforeEach(() => {
-      restore("postgres-writable");
-      cy.signInAsAdmin();
-    });
-    it("should not be possible to join with a table or question which is not in the same database (metabase#44974)", () => {
-      startNewQuestion();
-      entityPickerModal().within(() => {
-        entityPickerModalTab("Tables").click();
-        cy.findByText("Sample Database").click();
-        cy.findByText("Orders").click();
-      });
-
-      cy.button("Visualize").click();
-      cy.button("Save").click();
-      modal().button("Save").click();
+describe.skip("issue 44974", () => {
+  const PG_DB_ID = 2;
 
-      cy.wait(300);
-      modal().button("Not now").click();
+  beforeEach(() => {
+    restore("postgres-12");
+    cy.signInAsAdmin();
+  });
 
-      startNewQuestion();
-      entityPickerModal().within(() => {
-        entityPickerModalTab("Tables").click();
-        cy.findByText("Writable Postgres12").click();
-        cy.findByText("Scoreboard Actions").click();
-      });
+  it("should not be possible to join with a question from recents which is not in the same database (metabase#44974)", () => {
+    withDatabase(PG_DB_ID, ({ PEOPLE_ID }) => {
+      createQuestion(
+        {
+          name: "Question 44794 in Postgres DB",
+          query: {
+            database: PG_DB_ID,
+            "source-table": PEOPLE_ID,
+            limit: 1,
+          },
+        },
```

Comment:
The reason I like to always extract this into `questionDetails` is that you can then reference the `questionDetails.name` later in the assertion instead of typing it out again.

It also reduces the risk of a typo or strings being out of sync.